### PR TITLE
DIRSTUDIO-1300: Use LookupTranslator instead of chained replace() calls.

### PR DESCRIPTION
**Pull request by roubert**
https://issues.apache.org/jira/browse/DIRSTUDIO-1300

### CODE REVIEW

File: plugins/valueeditors/pom-first.xml [hunk 1]
```diff
@@ -64,10 +64,12 @@
  org.apache.directory.studio.valueeditors.uuid</Export-Package>
  
             <Import-Package>org.apache.commons.codec,
- org.apache.commons.codec.binary
+ org.apache.commons.codec.binary,
+ org.apache.commons.text
             </Import-Package>
             
             <Require-Bundle>
+ org.apache.commons.commons-text,
```
Comment (seelmann) [inline plugins/valueeditors/pom-first.xml:72]: Is the bundle required if the package is imported, or vice versa?

File: pom.xml [hunk 2]
```diff
@@ -137,8 +139,8 @@
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-compiler-plugin</artifactId>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <source>11</source>
+          <target>11</target>
```
Comment (seelmann) [inline pom.xml:143]: Thanks!

File: plugins/ldapbrowser.core/src/main/java/org/apache/directory/studio/ldapbrowser/core/jobs/ExportOdfRunnable.java [hunk 3]
```diff
@@ -60,6 +61,9 @@ public class ExportOdfRunnable implements StudioConnectionRunnableWithProgress
     /** The maximum count limit */
     public static final int MAX_COUNT_LIMIT = 65000;
 
+    /** The postal address decoder. */
+    private static CharSequenceTranslator DECODER = Utils.createPostalAddressDecoder( "\n" ); //$NON-NLS-1$;
```
Comment (seelmann) [inline plugins/ldapbrowser.core/src/main/java/org/apache/directory/studio/ldapbrowser/core/jobs/ExportOdfRunnable.java:65]: According to the javadoc all translators are thread-safe so the singleton is fine.

**Review by seelmann — APPROVED** [review]
LTGM

File: plugins/valueeditors/pom-first.xml (hunk 1 above)
Comment (roubert) [reply plugins/valueeditors/pom-first.xml:72]: I don't really understand how these dependency declarations work so what's here is all the result of trial-and-error (but the result is that both `build.sh` and Eclipse can build and run this the way it is now). If you or someone else actually knows how this really works, you might want to take a look afterwards and see if you can find things to correct or simplify.

Comment (seelmann) [reply plugins/valueeditors/pom-first.xml:72]: `Require-Bundle` means to use that specific bundle/dependency, and all packages in that bundle are imported.

`Import-Package` means to only import the declared packages, so it's more fine-grained. The actual implementation of the packagesis open at the `MANIFEST.MF` level. The pb is also that those declarations are not transitive, so I assume we need to import `org.apache.commons.text` and `org.apache.commons.text.translate`.

Fine to keep it like this, I'll have a look later if we can simplify.

**Pull request merged**